configure.ac

```diff
@@ -1093,6 +1095,8 @@ AC_DEFINE(COPYRIGHT_HOLDERS, "_COPYRIGHT_HOLDERS", [Copyright holder(s) before %
 AC_DEFINE(COPYRIGHT_HOLDERS_SUBSTITUTION, "_COPYRIGHT_HOLDERS_SUBSTITUTION", [Replacement for %s in copyright holders string])
 define(_COPYRIGHT_HOLDERS_FINAL, [patsubst(_COPYRIGHT_HOLDERS, [%s], [_COPYRIGHT_HOLDERS_SUBSTITUTION])])
 AC_DEFINE(COPYRIGHT_HOLDERS_FINAL, "_COPYRIGHT_HOLDERS_FINAL", [Copyright holder(s)])
+AC_DEFINE(CURRENT_RELEASE_DATE, "_CURRENT_RELEASE_DATE", [Date of the current release])
+AC_DEFINE(NEXT_RELEASE_DURATION, _NEXT_RELEASE_DURATION,[Days until the next release is definitely released])
```

Comment:
I was thinking that maybe "expected lifespan" covers the load here better?

Reply:
How does `EXPECTED_RELEASE_DURATION` read for you?

Reply:
That could fit to a warning message saying: "This release has been running for so long, either you forgot to update, developers are lazy, or the apocalypse happened".

Going to need some time to think about the meaning of this parameter a bit more.

Reply:
All three of those seem like things I'd want to remind myself to check personally every six or nine months anyhow.

Reply:
Given that it's an internal constant (and one we should document to update in the release process but won't expose outside of the core), it's as good as anything. I like its descriptiveness even though it's verbose. Let's see how other PRs built on it look.

Reply:
Back into the `std::chrono` mines for me! I'll update this now.